### PR TITLE
fix: lint

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -21,12 +21,12 @@ params {
     save_reference = null // Built references not saved
 
     // Main options
-    help = false
+    help = null
```
Comment (FriederikeHanssen) [inline nextflow.config:24]: Why is it better to use `null` than `false`?

Comment (maxulysse) [reply nextflow.config:24]: No idea, and probably not good anyway, since all tests are failing anyway

Comment (maxulysse) [reply nextflow.config:24]: ok, so looked into everything, ran everything once again locally, and found no issues.
So just relaunched the tests, and it's working...

File: nextflow.config [hunk 2]
```diff
@@ -18,15 +18,14 @@ params {
     igenomes_base = 's3://ngi-igenomes/igenomes/'
     igenomes_ignore = false
     genomes_base = null // Disabled by default
-    save_reference = null // Built references not saved
+    save_reference = false // Built references not saved
```
Comment (FriederikeHanssen) [inline nextflow.config:21]: sometimes it's false and sometimes null, what is the rule here

Comment (maxulysse) [reply nextflow.config:21]: So I choose to use `false` for boolean, and `null` for every other types

Comment (FriederikeHanssen) [reply nextflow.config:21]: ok, makes sense

File: nextflow_schema.json [hunk 3]
```diff
@@ -309,7 +309,7 @@
                 },
```
Comment (FriederikeHanssen) [inline nextflow_schema.json:309]: the other params don't need to be updated here?

Comment (maxulysse) [reply nextflow_schema.json:309]: No, all others were good already because of the json schema

Comment (maxulysse) [reply nextflow_schema.json:309]: issues where in the nextflow.config

**Pull request merged**